feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Timetable.kt

```diff
@@ -418,6 +417,21 @@ class ScreenScrollState(
         }
     }
 
+    suspend fun safeScroll(
```

Comment:
I didn't think it was our class. I think we can delete the `safeScroll` and use the implementation as the default implementation of the `scroll` method. Because it is scroll, so I think it can not be the critical feature.
What do you think?

But I think it is better than the current implementation. So please let me merge this for now!

Reply:
Thank you for your comment!
I've opened a PR for the rename and simplification https://github.com/DroidKaigi/conference-app-2022/pull/902